cryptography/src/lib.rs

```diff
@@ -122,8 +125,12 @@ pub trait BatchScheme {
     fn verify<R: RngCore + CryptoRng>(self, rng: &mut R) -> bool;
 }
 
-/// Byte array representing a hash digest.
-pub type Digest = Bytes;
+/// Error Digest manipulations must rely on.
+#[derive(Error, Debug)]
+pub enum DigestError {
```

Comment:
Let's call this `Error` instead of scoping this to `Digest`. If we migrate the `validate` functionality for `Scheme` in a similar way, we'll want to use a single error type in cryptography imo.